config/capsule-answers.yaml

```diff
@@ -0,0 +1,5 @@
+---
+certs:
+  generate: false
+  deploy: true
+capsule: true
```

Comment:
@mbacovsky, @ehelms, This is to follow-up to our discussion the other day.  The question was whether or not it is possible to sync a puppet repo (which is using the puppet install distributor) to a capsule, if puppet was not configured on the capsule.  I was finally able to run a test with upstream katello+capsule and the following scenario does generate an error to the user:
```
1. configure the capsule to sync the 'dev' environment
2. create a puppet repo & upload a module to it
3. create a content view, add the puppet module to it & publish to library
4. promote that content view to 'dev', which will trigger the content to be synced to the capsule
```

In the above scenario, the sync to the capsule will fail, since /etc/puppet/environments is non-existent.  The error indicates the following:
```
PLP0034: The distributor Default_Organization-dev-puppet_view indicated a failed response when publishing repository Default_Organization-dev-puppet_view.
```